.pre-commit-config.yaml

```
@@ -16,6 +16,21 @@ repos:
               | ^poetry/core/_vendor
           )
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.910
+    hooks:
+      - id: mypy
+        additional_dependencies:
+          - types-requests
+        files: |
+          (?x)(
+            ^poetry
+          )
+        exclude: |
+          (?x)(
+              ^poetry/core/_vendor
+          )
```

since we've moved our include/exclude rules to `pyproject.toml`, this can be simplified and closer aligned to the [config in poetry](https://github.com/python-poetry/poetry/blob/f7bbd54f5e543523d4d81221c9a8a4daee2e1315/.pre-commit-config.yaml#L43-L50)

```suggestion
    rev: v0.910-1
    hooks:
      - id: mypy
        pass_filenames: false
        additional_dependencies:
          - types-requests
```